velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -171,6 +171,91 @@ TEST_F(StringTest, conv) {
   EXPECT_EQ(conv("", 10, std::nullopt), std::nullopt);
 }
 
+TEST_F(StringTest, levenshtein) {
+  const auto levenshtein = [&](const std::optional<std::string>& left,
+                               const std::optional<std::string>& right) {
+    return evaluateOnce<int32_t>("levenshtein(c0, c1)", left, right);
+  };
+
+  const auto levenshteinWithThreshold =
+      [&](const std::optional<std::string>& left,
+          const std::optional<std::string>& right,
```

Comment:
Can we only use this one?
```
const auto levenshtein = [&](const std::optional<std::string>& left,
                               const std::optional<std::string>& right,
                               const std::optional<int32_t>& threshold,
                               int32_t levenshteinExpected,
                               int32_t levenshteinWithThresholdExpected) {
    EXPECT_EQ(
        evaluateOnce<int32_t>("levenshtein(c0, c1)", left, right),
        levenshteinExpected);
    EXPECT_EQ(
        evaluateOnce<int32_t>(
            "levenshtein(c0, c1, c2)", left, right, threshold),
        levenshteinWithThresholdExpected);
  };
```

Reply:
I got your point but I think this may not a good idea to mixed two functions together. for thresold function, there are some test cases we test for threshold > result, threshold = result, thresold < result. These may be duplicate for non-thresold one.

Reply:
do you have more comments around this? Thanks.

Reply:
Kindly ping, @jinchengchenghh , thanks.

Reply:
Let the community do the decision. I'm open to this.
Others look good to me

Reply:
Get, Updated, thanks.